gril/ril_constants.h

```diff
@@ -372,6 +372,17 @@
 #define RIL_REQUEST_VOICE_RADIO_TECH 108
 #define RIL_REQUEST_SET_INITIAL_ATTACH_APN 111
 
+#define RIL_REQUEST_IMS_REGISTRATION_STATE 112
+#define RIL_REQUEST_IMS_SEND_SMS 113
+#define RIL_REQUEST_GET_DATA_CALL_PROFILE 114
```

Comment:
GET_DATA_CALL_PROFILE doesn't appear to be a standard request defined by any version of AOSP's ril.h header.  If it's a custom OEM request, then it should be added in an OEM-specific header.  Also, as it's not used in any of your new code, I would just leave it out of this change.